lib/compress.js

```diff
@@ -3028,63 +3028,49 @@ merge(Compressor.prototype, {
         return this;
     });
 
-    var commutativeOperators = makePredicate("== === != !== * & | ^");
+    var reversibleOperators = makePredicate("+ * & | ^ < > <= >= == != === !==");
```

Comment:
+1 on the revert of https://github.com/mishoo/UglifyJS2/commit/88fb83aa813d38e21abeab2b5e4f74f02ef582c1